modules/rosa-hcp-sts-accessing-a-break-glass-cred-cli.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+//
+// * rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.adoc
+// * rosa_hcp/rosa-hcp-sts-creating-a-cluster-ext-auth.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-hcp-sts-accessing-a-break-glass-cred-cli_{context}"]
+= Accessing a {hcp-title} cluster by using a break glass credential
+
+Use the new `kubeconfig` from the break glass credential to gain temporary admin access to a {hcp-title} cluster.
+
+.Prerequisites
+
+* You have access to a {hcp-title} cluster with external authentication enabled. For more information, see _Creating {hcp-title} clusters with external authentication_.
+* You have installed the `oc` CLI.
+
+.Procedure
+
+. Optional: Create a `kubeconfigs` directory:
++
+[source,terminal]
+----
+$ mkdir ~/kubeconfigs
+----
++
+. Export the newly generated `kubeconfig` file, replacing <cluster_name> with the name of your cluster:
++
+[source,terminal]
+----
+$ export KUBECONFIG=~/kubeconfigs/break-glass-<cluster_name>.kubeconfig
+----
++
+. Access the details for the cluster:
++
+[source,terminal]
+----
+$ rosa describe break-glass-credential <break_glass_credential_id> -c <cluster_name>  --kubeconfig > $KUBECONFIG
+----
++
+. List the nodes from the cluster:
++
+[source,terminal]
+----
+$ oc get nodes
+----
+.Example output
++
+[source,terminal]
+----
+NAME                        STATUS   ROLES   AGE   VERSION
+ip-10-0-0-27.ec2.internal   Ready    worker  8m    v1.28.7+f1b5f6c
+ip-10-0-0-67.ec2.internal   Ready    worker  9m    v1.28.7+f1b5f6c
+----
+. Verify you have the correct credentials:
++
+[source,terminal]
+----
+$ oc auth whoami
```

Comment:
Should be kubectl - oc doesn't work here

![image](https://github.com/openshift/openshift-docs/assets/16167833/7bd3a5f0-8f66-4293-b97a-846dd625cca6)